### DISCUSSION TITLE
Is that Possible to Target Multiple Css Proptery in Style Manager ??

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5755] [General]
What i am Looking for to create a custom property  in style Manager
```
{
id: "size",
type: "size",
property: "size",
full: true,
}
```

Something like this the property should handle multiple css rule 
The type size will create a custom style property which creates 
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/3069f565-27d0-43e6-bff7-c126cbfc1194)
something like this the width handles max-width and height handles padding for top and bottom 

the function are working as expected but the issue when getting value from update function it return empty string (because its looking for the value of size but the size is not available in css rule so its returning empty ) how can i overcome this any possible solution

**Comment by artf** [comment]
The [composite type](https://grapesjs.com/docs/api/property_composite.html) has `fromStyle`/`toStyle` properties that allow you to do that

**Reply by RakulAgn** [reply]
I've created an addType method that allows for the addition of a new property. This method includes operations like create, emit, and update. Additionally, there's an addProperty method used for adding properties to existing sectors. This method also enables customization of the user interface based on those properties?. Just to clarify, both addType and addProperty are intended for introducing new properties, right?


and this is how i created with addType Method
https://gist.github.com/RakulAgn/52d40d78924366d1e5e41248220e9195

and i have added this
```
{
id: "size",
type: "size",
property: "size",
full: true,
},
```
to my html in stylemanager sectors
